### PR TITLE
[Improve Gearscript] Resupply crates now have space to drop stuff

### LINKED ISSUE #107: [Feature Request] Dynamic container load on gearscript
**Is your feature request related to a problem? Please describe.**
Overloading gearscript on a container or vehicle sucks cause it makes it harder to swap stuff around from your loadout.

**Describe the solution you'd like**
Use https://community.bistudio.com/wiki/setMaxLoad when a container or vehicle is gearscripted, to add 25-33% extra empty space on top of the resulting contents.

**Why is this this best way to solve the problem?**
cause its cool and new

**Additional context**
🤓

### PR DESCRIPTION
**Pull request by Joecuronium**
### Pull Request Description
**When merged this pull request will:**
Fix #107 
Make Gearscript automatically add 25% more load capacity over the assigned gear on all gearscripted crates
Not tested in MP but there is no way this won't work, right?

### Release Notes
Gearscript automatically adds 25% more load capacity over the assigned gear on all gearscripted crates

## IMPORTANT

- [ ] The Release Notes section below must be filled out appropriately to explain the changes made in this PR. This section will be used in Framework Changelogs.
- [ ] If the contribution affects [the wiki](https://github.com/CombinedArmsGaming/CAFE3/wiki), please include your changes in this pull request so the wiki is consistently updated.
- [ ] [Contribution Guidelines](https://github.com/CombinedArmsGaming/CAFE3/blob/release/contributing.md) are read, understood and applied.
- [ ] Title of this PR uses our standard template `[Descriptor] - Add|Fix|Improve|Change|Make|Remove {changes}`.

### CODE REVIEW

**Comment by Cre8or** [comment]
> Not tested in MP but there is no way this won't work, right?

We don't do that here.

I know it seems trivial, but please test it in MP regardless.

**Review by dgibso29 — CHANGES_REQUESTED** [review]
Requesting changes for improved test coverage

File: components/gearScript/fn_applyLoadout.sqf [hunk 1]
```diff
@@ -35,6 +35,10 @@ if (_typeofUnit find "crate_" == 0) exitWith
 
         } forEach _crateArray;
 
+        private _oldLoad = loadAbs _unit;
+
+        _unit setMaxLoad (_oldLoad * 1.25);
```
Comment (Bubbus) [inline components/gearScript/fn_applyLoadout.sqf]: Wiki says this command must be executed on the server, recommend using this - also untested though YMMV
```suggestion
        [_unit, (_oldLoad * 1.25)] remoteExecCall ["setMaxLoad", 2];
```

**Review by Bubbus — COMMENTED** [review]
Thanks for helping, it would have taken me a long time to get around to this.

**Comment by Joecuronium** [comment]
MP test done, protocols filled out in triplicate and filed with the arma commission, works as expected

File: components/gearScript/fn_applyLoadout.sqf (hunk 1 above)
Comment (Joecuronium) [reply components/gearScript/fn_applyLoadout.sqf]: already done during zeus module cleanup, commit references wrong issue tho

**Review by dgibso29 — APPROVED** [review]
🍰

**Pull request merged**